x-pack/test/fleet_api_integration/apis/index.js

```diff
@@ -4,9 +4,14 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export default function loadTests({ loadTestFile }) {
-  describe('Fleet Endpoints', () => {
+export default function ({ loadTestFile }) {
+  describe('Fleet Endpoints', function () {
+    this.tags('ciGroup7');
```

Comment:
ciGroup7 is already out longest CI Group, ciGroup4 is pretty short, want to move them there?

Reply:
Sure :)

Reply:
```suggestion
    this.tags('ciGroup10');
```